pkg/agent/controller/egress/egress_controller.go

```diff
@@ -147,6 +149,9 @@ type EgressController struct {
 	ipAssigner ipassigner.IPAssigner
 
 	egressIPScheduler *egressIPScheduler
+
+	meterTracker      map[uint32]*crdv1b1.RateLimit
+	meterTrackerMutex sync.RWMutex
```

Comment:
it seems you don't need a RWMutex since it's never read

Reply:
Thanks. Done.